backend/app/controllers/api/v1/questions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::QuestionsController < Api::V1::ApiController
+
+  before_action :set_question, only: [:show, :update, :destroy, :answer]
+
+  def index
+    @questions =
+      if question_params[:difficulty_level]
+        Question.filter(question_params[:difficulty_level])
+      else
+        Question.all
+      end
+      json_response(@questions)
```

Comment:
We don't need to nest this under `@questions`.  Please remove 2 spaces from left.